examples/next-prisma-websockets-starter/src/server/context.ts

```diff
@@ -11,8 +11,8 @@ import ws from 'ws';
  */
 export const createContext = async (
   opts:
-    | trpcNext.CreateNextContextOptions
-    | NodeHTTPCreateContextFnOptions<IncomingMessage, ws>,
+    | NodeHTTPCreateContextFnOptions<IncomingMessage, ws>
+    | trpcNext.CreateNextContextOptions,
```

Comment:
https://v6--typescript-eslint.netlify.app/rules/sort-type-constituents - is newly in the _`stylistic`_ config. While it does have an autofixer, I do worry that it's too nitpicky and will annoy people. Thoughts?

Reply:
If it doesn't have an autofixer it'll annoy the hell out of me :D

Reply:
It did break a thing so I had to disable it in a place: https://github.com/trpc/trpc/pull/4541/commits/794a2c8259853609ecf0a8b4f7f106fc2f6d74a2

Guessing this is if the two objects have the same property that conflicts or something?

Reply:
We could disable the rule until there's autofixing

Reply:
> We could disable the rule until there's autofixing?

There is autofixing - but the autofixer broke the type in one place. I'm guessing that's if you have something like 

```ts
type X = { foo: string }
type Y = { foo: number }
type Z = Y & X;
```

and it changes the order to `X & Y` instead

Reply:
...hmm. This smells like something we should fix in typescript-eslint-land. It's vaguely reminiscent of https://github.com/typescript-eslint/typescript-eslint/issues/6339.

@juliusmarminge are you up for filing an issue on us? If not no worries, I can - just I like more direct attribution for reporters. 😄